hello-bridge/DEPLOY.md

```diff
@@ -0,0 +1,242 @@
+# Deploying SimpleMinterUSC Contract
+
+This guide explains how to deploy the `SimpleMinterUSC` contract to Creditcoin USC networks. The contract serves as a bridge minter that verifies cross-chain proofs and mints tokens on Creditcoin.
+
+> [!NOTE]
+> **USC Testnet is coming soon!** Currently, only USC Devnet is available for deployment. Use Devnet for testing and development.
+
+## Prerequisites
+
+1. **Foundry installed**: Make sure you have Foundry installed. If not, install it:
+
+   ```bash
+   curl -L https://foundry.paradigm.xyz | bash
+   foundryup
+   ```
+
+2. **Funded wallet**: You need a wallet with native tokens (CTC) on the target network to pay for gas fees.
+   - Request test tokens from the [Creditcoin Discord faucet]
```

Comment:
Do we even have a Discord faucet for Devnet? If not, then people outside our team won't be able to get funds to run the tutorials.

I suppose it isn't worth setting up any extra infra since we'll be moving the tutorials to testnet soon anyways.

Reply:
No faucet, so this step needs some further documentation for devnet. Although, these examples should in the future only serve testnet. I think we need to add some TODO's to clean up some documentation that we are adding now